doc/nrf/ug_matter_hw_requirements.rst

```diff
@@ -0,0 +1,35 @@
+.. _ug_matter_hw_requirements:
+
+Matter hardware and memory requirements
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Hardware that runs Matter protocol applications must meet specification requirements, including providing the right amount of flash memory and being able to run Bluetooth LE and Thread concurrently.
+
+Supported SoCs
+**************
+
+Currently the following SoCs from Nordic Semiconductor are supported for use with the Matter protocol:
+
+* :ref:`nRF7002 <gs_programming_board_names>` (Matter over Thread and Matter over Wi-Fi)
```

Comment:
Is this link to SoC or to DK? If it's to DK, then the line 15 is misleading. We should list only SoCs here:

- nRF52840
- nRF5340
- nRF5340 + nRF7002